### PR TITLE
[External API Doc] Added rate limit endpoint section

### PR DESCRIPTION
**Pull request by jiepiao**
### Description

Added rate limit endpoint section.

### Screenshot
Screen Shot 2022-09-13 at 2 12 35 PM

Screen Shot 2022-09-13 at 2 13 14 PM

Screen Shot 2022-09-13 at 2 13 32 PM

### CODE REVIEW

**Comment by dachan** [comment]
@jiepiao can we also link to this from the [Rate Limits](https://api-docs.affinity.co/#rate-limits) section and add a mention of it in the [Changelog](https://api-docs.affinity.co/#changelog)?

**Comment by jiepiao** [comment]
> @jiepiao can we also link to this from the [Rate Limits](https://api-docs.affinity.co/#rate-limits) section and add a mention of it in the [Changelog](https://api-docs.affinity.co/#changelog)?

Sure, added link and updated changelog. I won't merge this until we enable the feature for all orgs.

**Review by dachan — APPROVED** [review]
Looks good to me, thanks!

**Comment by marcypark** [comment]
The rate-limit endpoint header says "whoami"

**Comment by jiepiao** [comment]
> The rate-limit endpoint header says "whoami"

Fixed and added monthly organization limit reset description.

File: source/includes/_changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2022-09-06
+
+- Added [Rate Limit](#rate-limit) documentation.
```
Comment (marcypark) [inline source/includes/_changelog.md:5]:
```suggestion
- Added [Rate Limit](#rate-limit) endpoint and documentation. Moved from a daily to a per minute per user limit. Monthly per account limits remain the same.
```

File: source/includes/_rate-limits.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 ## API Rate Limits
 
+For rate limit endpoint, please refer <a href="#rate-limit">Rate Limit</a>.
```
Comment (marcypark) [inline source/includes/_rate-limits.md:5]:
```suggestion
For our rate limit endpoint, please refer to <a href="#rate-limit-endpoint">Rate Limit endpoint</a>.
```

File: source/includes/_rate-limit-endpoint.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Rate Limit
+
+The rate limit endpoint allows you to see the monthly organization and per minute per api key limit usage.
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:3]:
```suggestion
The rate limit endpoint allows you to see your monthly account-level and per minute user-level API limits and usage.
```

File: source/includes/_rate-limit-endpoint.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+The monthly organization limit resets on at the end of each calendar month.
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:4]:
```suggestion
The monthly account-level call limit resets at the end of each calendar month.
```

File: source/includes/_rate-limits.md [hunk 5]
```diff
@@ -13,9 +15,9 @@ Current rate limits by plan tier are:
 | Premium       | 100,000         |
 | Enterprise    | Unlimited       |
```
Comment (marcypark) [inline source/includes/_rate-limits.md:16]: Add a line here too:

This monthly account-level limit resets at the end of each calendar month.

File: source/includes/_rate-limits.md [hunk 6]
```diff
@@ -2,6 +2,8 @@
 
 ## API Rate Limits
 
+For rate limit endpoint, please refer <a href="#rate-limit">Rate Limit</a>.
+
 ### Plan Tier Limits
 
 Your account plan tier will limit the number of requests each API key can make per month.
```
Comment (marcypark) [inline source/includes/_rate-limits.md:9]:
```suggestion
Your account plan tier will limit the overall number of requests you can make per month.
```

File: source/includes/_rate-limit-endpoint.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+## The Rate Limit Resource
+
+Querying the rate limit endpoint will give information about organization and api key level rate limit usage.
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:8]:
```suggestion
The rate limit resource includes information about account (AKA organization)-level and API key-level rate limits and usage.
```

File: source/includes/_rate-limit-endpoint.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+            "used": 0
+        }
+    }
+}
+```
+| Attribute          | Type    | Description                                  |
+| ------------------ | ------- | -------------------------------------------  |
+| rate               | object  | Type of rate limit.                          |
+| org_monthly        | object  | Monthly rate limit per organization.         |
+| api_key_per_minute | object  | Per minute rate limit per api key.           |
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:34]:
```suggestion
| api_key_per_minute | object  | Per minute rate limit per API key.           |
```

File: source/includes/_rate-limit-endpoint.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+| limit              | integer | The maximum number of calls for each period. |
+| remaining          | integer | The number of calls remaining before reset.  |
+| reset              | integer | Time in seconds until next time limit reset. |
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:37]:
```suggestion
| reset              | integer | Time in seconds until call count resets. |
```

File: source/includes/_rate-limit-endpoint.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+| used               | integer | The number of calls that have been used.     |
+
+<aside class="notice">
+  <h6>Note</h6>
+  <ul>
+    <li><code>/rate-limit</code> and <code>/auth/whoami</code> endpoints are excluded from rate limiting rules.</li>
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:43]:
```suggestion
    <li><code>/rate-limit</code> and <code>/auth/whoami</code> endpoints are exempt from organization-level monthly rate limits.</li>
```

File: source/includes/_rate-limit-endpoint.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+  </ul>
+</aside>
+
+## Get Rate Limit
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:47]:
```suggestion
## Get Rate Limit Information
```

File: source/includes/_rate-limit-endpoint.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+            "reset": 0,
+            "used": 0
+        }
+    }
+}
+```
+
+`GET /rate-limit`
+
+Gets information about the organization and api key level rate limit usage.
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:78]:
```suggestion
Querying the rate limit endpoint will yield information about account (AKA organization)-level and API key-level rate limits and usage.
```

File: source/includes/_rate-limit-endpoint.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+
+### Returns
+
+JSON body of data including limit, remaining, reset and used call number.
```
Comment (marcypark) [inline source/includes/_rate-limit-endpoint.md:82]:
```suggestion
The rate limit resource, a JSON body of data including limits, calls remaining, seconds until reset and call count.
```

**Review by marcypark — COMMENTED** [review]
Took the chance to take a look over the whole PR -- Lmk if you'd like to review these on Zoom! It should hopefully be easy just to merge my suggestions straight in.

**Pull request merged**